src/main/java/org/fundacionjala/pivotal/ui/pages/Dashboard.java

```diff
@@ -110,4 +123,20 @@ public String getMessageDeleteWorkspace() {
     public void refreshPage() {
         driver.navigate().refresh();
     }
+
+    /**
+     * This method click in config icon of Workspace.
+     * @param nameProject is a string with name of workspace
+     * @return a SettingWorkspace page object.
+     */
+    public SettingWorkspace clickConfigIconWorkSpace(final String nameProject) {
+        final String selector = String.format("//a[text()=('%s')]/following-sibling::span/a", nameProject);
+        By bLocator = new By.ByXPath(selector);
+        WebElement wConfigBtn = DriverManager
+                .getInstance()
+                .getDriver()
```

Comment:
this class extends from AbstractBasePage class
so It is not necessary to use the DriverManager you can use the driver variable directly from AbstractBasePage class